files/en-us/web/api/mouseevent/initmouseevent/index.html

```diff
@@ -16,19 +16,21 @@
 <p>The <code><strong>MouseEvent.initMouseEvent()</strong></code> method initializes the
   value of a mouse event once it's been created (normally using the {{domxref("Document.createEvent()")}} method).</p>
 
-<p>Events initialized in this way must have been created with the {{domxref("Document.createEvent()") }} method.
-  This method must be called to set the event
-  before it is dispatched, using {{ domxref("EventTarget.dispatchEvent()") }}. Once
-  dispatched, it doesn't do anything anymore.</p>
+<div class="notecard warning">
 
-<div class="notecard note">
-  <h4>Note</h4>
   <p><strong>Do not use this method anymore as it is deprecated.</strong></p>
 
   <p>Instead use specific event constructors, like {{domxref("MouseEvent.MouseEvent", "MouseEvent()")}}.
   The page on <a href="/en-US/docs/Web/Events/Creating_and_triggering_events">Creating and triggering events</a> gives more information about the way to use these.</p>
 </div>
 
+<p>Events initialized in this way must have been created with the {{domxref("Document.createEvent()") }} method.
+  This method must be called to set the event
+  before it is dispatched, using {{ domxref("EventTarget.dispatchEvent()") }}. Once
+  dispatched, it doesn't do anything anymore.</p>
```

Comment:
```suggestion
  This method must be called to set the event
  before it is dispatched, using {{ domxref("EventTarget.dispatchEvent()") }}.</p>
```
Shall we just drop that sentence? It seems unnecessary and it’s not completely clear to me what it’s trying to say.

Reply:
Yes!